captum/metrics/_core/infidelity.py

```diff
@@ -345,6 +346,15 @@ def infidelity(
                 `input batch size * n_perturb_samples`.
 
                 Default: None
+        normalize (bool, optional): Normalize the dot product of the input
+                perturbation and the attribution so the infedelity value is invariant
+                to constant scaling of the attribution values. The normalization factor
+                is defined as the ratio of two mean values across all perturbations:
+                `mean(dot product * func value diff) / mean(dot product * dot product)`.
```

Comment:
nit: Would it be possible to make this line a little more detailed to explain this factor for new users? E.g. it may not be immediately clear that func value diff is the same as the difference between the predictor function at its input and perturbed input described above.

Reply:
sure, i can use latex here